frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelistedHealthChecks.ts

```diff
@@ -0,0 +1,6 @@
+export const whitelistedHealthChecks = new Set(['MON_DISK_LOW', 'MON_DOWN']);
+
+export const whitelistedHealthChecksRef = {
+  MON_DISK_LOW: 'https://www.example.com',
+  MON_DOWN: 'https://www.example.com',
```

Comment:
We should add doc links here. We can ask to create a kcs doc that can be updated later.

Reply:
KCS docs are alive and not updated every release like product docs.

Reply:
This might not be a KCS, and most probably will be a documentation link. Once I have the confirmation, I will let you know